cmd/main.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
I can't, otherways, it will basically not compile sice you can only have one package under the same folder and also, to have a main, you need to be outside the `pkg` directory and inside a `cmd` one. 

I asked @julesdesmit and seems that this was the only solution. 
What I can do is basically remove the `main.go` file and just use it locally to test the protocol.